### ISSUE TITLE
Potential chrome web security issue in chromium based browsers

### ISSUE THREAD

**Issue by JHuangg** [cypress-io/cypress#17770]
### Current behavior

![chrome2](https://user-images.githubusercontent.com/77698733/129744132-3fdf32cd-2fef-4f81-8a7b-b320b9cf4291.png)
![chromewebsecurity](https://user-images.githubusercontent.com/77698733/129744134-ae43a7ba-749c-43f4-86cd-d053ed84b80d.png)

Currently, I have chromeWebSecurity set to false in my cypress.json, however whenever I run tests with chrome, it's being detected as true.

### Desired behavior

Cypress should be able to visit external links with chromeWebSecurity set to false.

### Test code to reproduce

The bug comes from code that redirects the user to the Google Login URL. 

### Cypress Version

8.3.0

**Comment by jennifer-shehane** [comment]
You have `chromeWebSecurity` set to `false` and the resolved configuration in your screenshot shows that it is set to `false`. What makes you believe it is being evaluated as `true`?

**Comment by JHuangg** [comment]
> You have `chromeWebSecurity` set to `false` and the resolved configuration in your screenshot shows that it is set to `false`. What makes you believe it is being evaluated as `true`?

The first image is my Chrome browser telling me that I am trying to redirect to a new URL, which is true as I am trying to redirect from my test website to Google.com. I am able to redirect just fine on the electron browser, however on the chrome browser, it's telling me that I'm not allowed to redirect unless chromeWebSecurity is set to false. I have attached the second screenshot to show that I have configured this setting to false already in cypress.json, however its still returning as if it was true.
